frontend/kesaseteli/employer/src/components/application/form/Checkbox.tsx

```diff
@@ -0,0 +1,73 @@
+import { Checkbox as HdsCheckbox } from 'hds-react';
```

Comment:
checkbox is for the terms and conditions:
![Screenshot 2021-09-23 at 11 03 53 AM](https://user-images.githubusercontent.com/7648194/134473375-5d8f0288-1866-49b1-b90f-16ffa5e7fa28.png)